cli/pkg/kctrl/cmd/package/repository/release/release.go

```diff
@@ -165,14 +165,16 @@ func (o *ReleaseOptions) Run() error {
 			ImgLockFilepath:   tempImgpkgLockPath,
 			UI:                o.ui,
 		}
-		bundleURL, err = imgpkgRunner.Run()
+		bundleURLWithSHARef, err = imgpkgRunner.Run()
 		if err != nil {
 			return err
 		}
 	}
 
 	artifactWriter := NewArtifactWriter(pkgRepoName, wd)
-	err = artifactWriter.WritePackageRepositoryFile(bundleURL)
+	repoURL := strings.Split(bundleURLWithSHARef, "@sha")[0]
```

Comment:
Should we separate it on the basis of `@` or `@sha`. I think `@` is not a valid character for repo name. But not able to find concrete documentation